WooCommerce/Classes/ViewRelated/Upgrades/UpgradesView.swift

```diff
@@ -39,30 +44,57 @@ struct UpgradesView: View {
     }
 
     var body: some View {
-        List {
-            Section {
+        VStack {
+            VStack(alignment: .leading, spacing: Layout.contentSpacing) {
                 Text(planText)
                 Text(daysLeftText)
             }
-            Section {
-                VStack {
-                    Image(uiImage: .emptyOrdersImage)
-                    if let availableProduct = upgradesViewModel.retrievePlanDetailsIfAvailable(.essentialMonthly) {
-                        Text(availableProduct.displayName)
-                            .font(.title)
-                        Text(Constants.upgradeSubtitle)
-                            .font(.body)
-                        Text(availableProduct.displayPrice)
-                            .font(.title)
-                    }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(.leading)
+
+            Spacer()
+
+            Image(uiImage: upgradesViewModel.userIsAdministrator ? .emptyOrdersImage : .noStoreImage)
```

Comment:
Thoughts for a future change – perhaps it would be simpler/more readable to define two views, one for owners and one for everyone else, then switch at the top level? 

I'll keep this in mind as I move on to the UI